src/backends/torch/torchlib.cc

```diff
@@ -362,6 +442,7 @@ namespace dd
 
     if (!_classif_layer_file.empty())
       {
+        _logger->info("loading " + _classif_layer_file);
         torch::load(_classif, _classif_layer_file, _device);
```

Comment:
Model is loaded here as well (This method is called during training to finalize the classification layer with some example data)
It would be great to replace torch::load with classif_model_load

Reply:
ok done